.envrc

```diff
@@ -1,2 +1,4 @@
-watch_file *.nix
+watch_file *.nix **/*.nix
```

Comment:
Inclusion of the nix-option-search.sh does make sense.

But does "**/*.nix" really work? The "**" does not seem to work for me.
It would be nice to have a simple, recursive glob for all nix files, though.

Reply:
You are right, it doesn't work the way I put it in here originally. [`watch_file`](https://github.com/direnv/direnv/blob/bbffa7241bd215f0f48db550ebac9e755a60838b/stdlib.sh#L416) uses bash globs to evaluate this. By using the `globstar` shell option, it does what it is supposed to do:

> If set, the pattern ‘**’ used in a filename expansion context will match all files and zero or more directories and subdirectories. If the pattern is followed by a ‘/’, only directories and subdirectories match.